### PR TITLE
feat: add new package 'tonetwistplugs'

### PR DESCRIPTION
**Pull request by SpotlightKid**

### CODE REVIEW

File: packages/tonetwistplugs/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  "dpf-$_dpf_commit.tar.gz::https://github.com/DISTRHO/DPF/archive/$_dpf_commit.tar.gz"
+  "pugl-$_pugl_commit.tar.gz::https://github.com/DISTRHO/pugl/archive/$_pugl_commit.tar.gz"
+)
+sha256sums=('99190f9ddc03eb8586abdf8afcaa8ec8ecf81db7af3db7791949b5702ff3df33'
+            '5e7cf7fef94240420f345fcf68bf7c920e88256a11ea3ae264f55644a2fb9995'
+            'd83e958d8d699d60c2425eb51de7b38244b3f86e7a84ede83d6454b84cbf7ce5')
+_plugins=(
+  BoobTube
+  CollisionDrive
+  MetalTone
```
Comment (cbix) [inline packages/tonetwistplugs/PKGBUILD:36]: Oh, just noticed this after approving the 2 other PRs. Doesn't it conflict with the other packages?

**Review by cbix — CHANGES_REQUESTED** [review]

File: packages/tonetwistplugs/PKGBUILD (hunk 1 above)
Comment (SpotlightKid) [reply packages/tonetwistplugs/PKGBUILD:36]: No, two different implementations of the plugin with different LV2 URIs and installation dir.

Comment (SpotlightKid) [reply packages/tonetwistplugs/PKGBUILD:36]: * `/usr/lib/lv2/collisiondrive.lv2` -> `urn:brummer10:collisiondrive`
* `/usr/lib/lv2/CollisionDrive.lv2` -> `https://github.com/brummer10/CollisionDrive`

Comment (cbix) [reply packages/tonetwistplugs/PKGBUILD:36]: Indeed, I see no collision here 🥁

**Pull request merged**